polaris_wyre/wyre/__init__.py

```diff
@@ -0,0 +1 @@
+from .wyre import Wyre
```

Comment:
Why is it needed?

Reply:
Because of the import of `Wyre` class will be easier: `from polaris_wyre.wyre import Wyre`. Otherwise, it would be: `from polaris_wyre.wyre.wyre import Wyre`.